### PR TITLE
Migration to TeamsFx Toolkit Samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Review by Prithvi-MSFT — APPROVED** [review]
changes look fine

File: samples/bot-sso-adaptivecard/nodejs/build/aad.manifest.dev.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
+{
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/bot-sso-adaptivecard/nodejs/build/aad.manifest.dev.json]: Fixed

**Review by formulahendry — APPROVED** [review]

**Comment by ChetanSharma-msft** [comment]
@Harikrishnan-MSFT - Run build pipeline and share the status.

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5641334002

image

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**